js/addToTask.js

```diff
@@ -58,9 +63,12 @@ ADD_TASK.addEventListener(CLICK_EVENT, function (event) {
       done: false,
       edit: false,
       trash: false,
+      startDate: taskStartDate,
```

Comment:
why not store the timestamp directly if there is no other use of task start date as string?

```
      startDate: Date.now(),
```